less/forum.less

```diff
@@ -15,6 +15,14 @@
 }
 .SubscriptionMenu .Dropdown-menu {
   min-width: 260px;
+
+  @media @tablet-up {
+    & li:first-child {
```

Comment:
I believe `& li` is identical to just using `li` here?

Reply:
yeah it's just a bad habit, I do this all the time